gr-fec/lib/polar_decoder_common.cc

```diff
@@ -168,11 +168,11 @@ void polar_decoder_common::print_pretty_llr_vector(const float* llr_vec) const
 {
     for (int row = 0; row < block_size(); row++) {
         // FIXME this is an interesting mixture of iostream and stdio
```

Comment:
You'd want to remove this comment once it's consistently doing `fmt::print`, as well!

Reply:
Ok, then refactoring to `fmt::print`

Reply:
thank you so much :)